apps/condo/domains/resident/schema/Resident.js

```diff
@@ -191,7 +193,7 @@ const Resident = new GQLListSchema('Resident', {
     plugins: [uuided(), versioned(), tracked(), softDeleted(), historical()],
     hooks: {
         validateInput: async ({ resolvedData, operation, addValidationError, context }) => {
-            const { address, addressMeta, unitName, unitType, user: userId } = resolvedData
+            const { address, addressMeta, unitName, user: userId } = resolvedData
             if (operation === 'create') {
                 const addressUpToBuilding = getAddressUpToBuildingFrom(addressMeta)
                 const [resident] = await ResidentAPI.getAll(context, {
```

Comment:
you've removed `unitType`, but it's still being used

Reply:
missed on rebase, fixed